BE/src/main/java/com/sidedish/repository/ItemRepository.java

```diff
@@ -4,13 +4,21 @@
 import com.sidedish.domain.Item;
 
 import org.springframework.data.jdbc.repository.query.Query;
-import org.springframework.data.repository.CrudRepository;
+import org.springframework.data.repository.PagingAndSortingRepository;
 import org.springframework.data.repository.query.Param;
 
+import org.springframework.data.domain.PageRequest;
+import java.util.List;
 import java.util.Set;
 
-public interface ItemRepository extends CrudRepository<Item, Long> {
+public interface ItemRepository extends PagingAndSortingRepository<Item, Long> {
 
     @Query("SELECT i.* FROM items as i join item_order as io on i.id = io.item where io.orders = :id")
     Set<Item> findByOrderId(@Param("id") Long id);
+
+    List<Item> findByCategory(Long typeNumber, PageRequest pageable);
```

Comment:
이건 참고차, Page 타입으로도 받을 수 있습니다.

```suggestion
    Page<Item> findByCategory(Long typeNumber, PageRequest pageable);
```